src/main/java/net/snowflake/client/jdbc/cloud/storage/SnowflakeAzureClient.java

```diff
@@ -118,7 +118,7 @@ private void setupAzureClient(
       }
       this.azStorageClient = new CloudBlobClient(storageEndpoint, azCreds);
       opContext = new OperationContext();
-      HttpUtil.setProxyForAzure(opContext);
+      HttpUtil.setProxyForAzure(stage.getProxyProperties(), opContext);
```

Comment:
Same question, is that possible to utilize the sfSession here?

Reply:
Same as above answer ^